config.md

```diff
@@ -308,15 +308,26 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps where both the key and value MUST be strings.
+While the key and value MUST be present, they MAY be empty strings.
+Keys MUST be unique, and best practice is to namespace the keys.
+Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myK
+ey`.
+Keys starting with the `org.opencontainers` namespace are reserved and MUST NOT be used.
+If there are no annotations then this property MAY either be absent or an empty map.
+Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.
 
 ```json
 "annotations": {
-    "key1" : "value1",
-    "key2" : "value2"
+    "example.com/gpu-cores" : "2"
```

Comment:
This needs to get reversed too, and I guess it would be `com.example.gpu-cores`.  The slash approach doesn't seem as appropriate with the reversed domain names.